{"name": "Search for a product in ebay home page", "status": "broken", "statusDetails": {"message": "KeyError: 'f6d5f6d2-ad48-4cc5-9a66-ef4ab3bd3e23'\n", "trace": "  File \"C:\\Python310\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python310\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_newsearch.py\", line 20, in step_impl\n    allureLogs(\"Search button is clicked\")\n  File \"features\\steps\\test_newsearch.py\", line 31, in allureLogs\n    with allure.step(text):\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\_allure.py\", line 182, in __exit__\n    plugin_manager.hook.stop_step(uuid=self.uuid, title=self.title, exc_type=exc_type, exc_val=exc_val,\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 113, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Python310\\lib\\site-packages\\allure_behave\\listener.py\", line 163, in stop_step\n    self.logger.stop_step(uuid,\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 139, in stop_step\n    self._update_item(uuid, **kwargs)\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 62, in _update_item\n    item = self._items[uuid] if uuid else self._items[next(reversed(self._items))]\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 33, in __getitem__\n    return self.thread_context.__getitem__(item)\n"}, "steps": [{"name": "Given the user is on the ebay home page", "status": "passed", "steps": [{"name": "Given the user is on the ebay home page", "status": "passed", "start": 1728969040108, "stop": 1728969040108}], "start": 1728969040108, "stop": 1728969040108}, {"name": "When the search text and make year is entered", "status": "passed", "steps": [{"name": "When the search text and make year is entered", "status": "passed", "attachments": [{"name": ".table", "source": "5960dfe7-ad3c-4a9a-a3ce-5f46784dfd67-attachment.csv", "type": "text/csv"}], "start": 1728969040111, "stop": 1728969040113}], "attachments": [{"name": ".table", "source": "f18fb304-2a2a-477f-915b-356303d4b3ad-attachment.csv", "type": "text/csv"}], "start": 1728969040108, "stop": 1728969040113}, {"name": "And the search button is clicked", "status": "broken", "statusDetails": {"message": "KeyError: 'f6d5f6d2-ad48-4cc5-9a66-ef4ab3bd3e23'\n", "trace": "  File \"C:\\Python310\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python310\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_newsearch.py\", line 20, in step_impl\n    allureLogs(\"Search button is clicked\")\n  File \"features\\steps\\test_newsearch.py\", line 31, in allureLogs\n    with allure.step(text):\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\_allure.py\", line 182, in __exit__\n    plugin_manager.hook.stop_step(uuid=self.uuid, title=self.title, exc_type=exc_type, exc_val=exc_val,\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 113, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Python310\\lib\\site-packages\\allure_behave\\listener.py\", line 163, in stop_step\n    self.logger.stop_step(uuid,\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 139, in stop_step\n    self._update_item(uuid, **kwargs)\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 62, in _update_item\n    item = self._items[uuid] if uuid else self._items[next(reversed(self._items))]\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 33, in __getitem__\n    return self.thread_context.__getitem__(item)\n"}, "steps": [{"name": "And the search button is clicked", "status": "broken", "statusDetails": {"message": "KeyError: 'f6d5f6d2-ad48-4cc5-9a66-ef4ab3bd3e23'\n", "trace": "  File \"C:\\Python310\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"C:\\Python310\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\test_newsearch.py\", line 20, in step_impl\n    allureLogs(\"Search button is clicked\")\n  File \"features\\steps\\test_newsearch.py\", line 31, in allureLogs\n    with allure.step(text):\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\_allure.py\", line 182, in __exit__\n    plugin_manager.hook.stop_step(uuid=self.uuid, title=self.title, exc_type=exc_type, exc_val=exc_val,\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_hooks.py\", line 493, in __call__\n    return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_manager.py\", line 115, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 113, in _multicall\n    raise exception.with_traceback(exception.__traceback__)\n  File \"C:\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Python310\\lib\\site-packages\\allure_behave\\listener.py\", line 163, in stop_step\n    self.logger.stop_step(uuid,\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 139, in stop_step\n    self._update_item(uuid, **kwargs)\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 62, in _update_item\n    item = self._items[uuid] if uuid else self._items[next(reversed(self._items))]\n  File \"C:\\Python310\\lib\\site-packages\\allure_commons\\reporter.py\", line 33, in __getitem__\n    return self.thread_context.__getitem__(item)\n"}, "steps": [{"name": "Search button is clicked", "steps": [{"name": "Search button is clicked", "status": "passed", "start": 1728969040114, "stop": 1728969040114}], "start": 1728969040114}], "start": 1728969040113, "stop": 1728969040125}], "start": 1728969040113, "stop": 1728969040123}, {"name": "Then the search results page is displayed", "status": "skipped", "start": 1728969040126, "stop": 1728969040126}, {"name": "Then the search results page is displayed", "status": "skipped", "start": 1728969040129, "stop": 1728969040130}], "start": 1728969040107, "stop": 1728969040130, "uuid": "00844a6c-2eb0-48ab-8c98-76dfaf25e9e0", "historyId": "2c87265c0b7414655403f98678c6e346", "fullName": "search functionality: Search for a product in ebay home page", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "search functionality"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}